lib/index.cjs

```diff
@@ -22,7 +22,15 @@ var Downloader = require("./downloader.cjs");
 
 var pkg = require("../package.json");
 
-updateNotifier({ pkg }).notify();
+(async () => {
+  let updateNotifier = undefined;
+  try {
+    updateNotifier = await import('update-notifier');
+    updateNotifier({ pkg }).notify();
+  } catch(e) {
+    console.error(e);
+  }
+})
```

Comment:
if you end up going this route, remove the manifest comment from the `package.json` too

Reply:
Done!